Content.Shared/Damage/DamageContainer/DamageContainerPrototype.cs

```diff
@@ -31,18 +32,24 @@ public virtual void LoadFrom(YamlMappingNode mapping)
             var serializer = YamlObjectSerializer.NewReader(mapping);
 
             serializer.DataField(ref _id, "id", string.Empty);
+            serializer.DataField(ref _supportAll, "supportAll", false);
             serializer.DataField(ref _supportedClasses, "supportedClasses", new HashSet<DamageClass>());
             serializer.DataField(ref _supportedTypes, "supportedTypes", new HashSet<DamageType>());
 
-            var originalTypes = _supportedTypes.ToArray();
+            if (_supportAll)
+            {
+                _supportedClasses.UnionWith(Enum.GetValues<DamageClass>());
+                _supportedTypes.UnionWith(Enum.GetValues<DamageType>());
+                return;
+            }
 
             foreach (var supportedClass in _supportedClasses)
             foreach (var supportedType in supportedClass.ToTypes())
```

Comment:
![image](https://user-images.githubusercontent.com/31366439/101493840-ae8b5800-39ba-11eb-823a-81eee6f28372.png)

Reply:
The nested foreach's have been defeated...